ans/ANS-104.md

```diff
@@ -221,6 +226,21 @@ A tag object is valid iff.:
 To write a bundle of DataItems, each DataItem should be constructed, signed, encoded, and placed in a transaction with
 the transaction body format and transaction tags specified in Section 1.
 
+#### 3.1 Nested bundle
+
+It is possible to construct data-items in such a way that they themselves are a bundle.
```

Comment:
Nit: data-items are referred to as `DataItems` everywhere else in the doc.

Reply:
I'd suggest reorganizing this section a bit. Consider something like:
```
Arweave Transactions and DataItems have analogous specifications for tagging and bearing of a binary payload. As such, the ANS-104 Bundle Transaction tagging and binary data format specification can be applied to the tags and binary data payload of a DataItem. Assembling a DataItem this way provides for the nesting of ANS-104 Bundles with one-to-many relationships between "parent" and "child" bundles and theoretically unbounded levels of nesting. Additionally, nested DataItem Bundles can be mixed heterogeneously with non-Bundle DataItems at any depth in the Bundle tree.

To construct an ANS-104 DataItem as a nested Bundle:
- Add tags to the DatItem as described by the specification in [section 1.1](#### 1.1 Transaction Tags)
- Provide a binary payload for the DataItem matching the Bundle Transaction Body Format described in [section 1.2](#### 1.2 Transaction Body Format), i.e. the Bundle header outlining the count, size, and IDs of the subsequent, nested DataItems, each of which should be verifiable using the method described in [section 2.1](#### 2.1 Verifying a DataItem).

Gateway GQL queries for of DataItem headers should, upon request, contain a `bundledIn` field whose value indicates
the parent-child relationship of the DataItem to its immediate parent. Any nested bundle should be traceable
to a base layer Arweave Transaction by recursively following the bundledIn field up through the chain of parents.
```

Reply:
I fixed one DatItem -> DataItem and applied your changes.